src/savegame_json.cpp

```diff
@@ -679,6 +743,17 @@ void player::store( JsonOut &json ) const
     }
     json.end_array();
 
+    json.member( "automoveroute" );
+    json.start_array();
+    for( const auto &elem : auto_move_route ) {
```

Comment:
please remove this auto
additionally, if it's a supported type, `json.member( "automoveroute", auto_move_route )` should work rather than writing the for loop here.